### PR TITLE
Fix broken CardTemplate

### PR DESCRIPTION
**Pull request by Ron-Lavi**
Using CardTemplate from Foreman in foreman_puppet v4.0.3, led to the puppet tab not functioning.

### CODE REVIEW

**Comment by Ron-Lavi** [comment]
@Manisha15 can you take a look? CI is green

File: .github/workflows/ruby_tests.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
       fail-fast: true
       matrix:
         foreman-repo: [theforeman/foreman]
-        foreman-core-branch: [develop]
+        foreman-core-branch: [3.3-stable]
```
Comment (kamils-iRonin) [inline .github/workflows/ruby_tests.yml:40]: What do you think about testing both compatible versions of Foreman (3.3 and 3.4)?

```suggestion
        foreman-core-branch: [3.3-stable, 3.4-stable]
```

Comment (Ron-Lavi) [reply .github/workflows/ruby_tests.yml:40]: Sounds good, I will add it

Comment (Ron-Lavi) [reply .github/workflows/ruby_tests.yml]: the test failed for foreman 3.4 stable due to the graphql issue.
Is it safe to cherry-pick #322 into our foreman_puppet 4.0-stable branch which also goes with foreman 3.3?
or should we split the branch at this point into:
4.0-stable => foreman 3.3
4.1-stable => foreman 3.4

what do you think @Manisha15 @kamils-iRonin?

Comment (kamils-iRonin) [reply .github/workflows/ruby_tests.yml]: I opened PR against `4.0-stable` branch https://github.com/theforeman/foreman_puppet/pull/324

**Comment by Ron-Lavi** [comment]
Updated the CI to include only `foreman 3.3-stable`,
and once it will be merged I will split this branch to:
```
4.0-stable => foreman 3.3
4.1-stable => foreman 3.4
```

**Review by Manisha15 — APPROVED** [review]

**Pull request merged**